web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig

```diff
@@ -0,0 +1,122 @@
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+] %}
+{{ attach_library('classy/node') }}
+<article{{ attributes.addClass(classes) }}>
+
+  {{ title_prefix }}
+  {% if label %}
+    <h1{{ title_attributes }}>
+      <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+    </h1>
+  {% endif %}
+  {{ title_suffix }}
+  <div class="row">
+    <div{{ content_attributes.addClass('node__content', 'inner-content') }}>
+      {{ content.field_r_hero_image }}
+      <div class="textbox-container">
+        <p>Cooking time: {{ content.field_r_cooking_time }}</p>
+      </div>
+      <div class="textbox-container">
+        {{ content.field_r_serves }}
+      </div>
+      <div class="textbox-container">
+        {{ content.field_r_cost }}
+      </div>
+      <div class="inner-content">
+        {{ content.field_r_summary }}
+      </div>
+      {{ content.field_r_body }}
+      {{ content.field_r_categories }}
+    </div>
+    <aside class="recipe-sidebar">
+      <div class="recipe-author-container">
+        <div class="recipe-author-text-container">
```

Comment:
Delete the existing Profile Image field for peoples accounts and recreate it using the "media" reference not image and ensure you add the author image here as per designs.

Reply:
done